CHANGELOG.md

```diff
@@ -45,6 +45,8 @@
 - [420](https://github.com/vegaprotocol/vegawallet/pull/420) - Verify initialisation state on service run
 - [421](https://github.com/vegaprotocol/vegawallet/pull/421) - Improve help texts
 - [423](https://github.com/vegaprotocol/vegawallet/pull/423) - Support launching the token dApp in `service run` command
+- [427](https://github.com/vegaprotocol/vegawallet/pull/427) - Sign MacOS binaries
+- [444](https://github.com/vegaprotocol/vegawallet/pull/444) - Fail gracefully when trying to get password input on msys
```

Comment:
This is at the wrong place, it should be under `Unreleased`

Reply:
My bad, will fix